Scripts/Items/Tools/FishingPole.cs

```diff
@@ -225,13 +225,12 @@ public int GetStrRequirement()
 
         public void OnFishedHarvest(Mobile from, bool caughtAnything)
         {
-            if (m_HookType == HookType.None)
-            {
-                from.SendLocalizedMessage(1149854); //As the magic of the hook fades, it transforms to a normal fishhook.  The fishing pole returns to normal.
-            }
-            else
+            if (m_HookType != HookType.None)
             {
                 HookUses--;
+
+                if (m_HookType == HookType.None)
```

Comment:
can never use it?

Reply:
Yes you can. HookType.NOne is default with no hook. With your change (before i reverted), it was spamming the message over and over. HookUses, if set to zero, sets it to hooktype.None. If this cloud shit cant recognize this, we shouldn't have it integrated. This is causing more problems than fixing.